packages/monaco/src/browser/monaco-frontend-module.ts

```diff
@@ -15,6 +15,7 @@
 // *****************************************************************************
 
 import * as MonacoNls from '@theia/monaco-editor-core/esm/vs/nls';
+import './monaco-monkey-patches';
```

Comment:
The reason why the `MonacoNls` override stuff happens before any other monaco code is loaded, is that the `nls.localize` method is often times called during load-time of code.

I'm not sure whether `vs/editor/common/languages` or `vs/editor/common/languages/supports/tokenization` use localizations at load-time, but just to be sure, please move this import after the `Object.assign` call.